operator/config/manifests/bases/rhacs-operator.clusterserviceversion.yaml

```
@@ -12,6 +12,8 @@ metadata:
     operatorframework.io/suggested-namespace: rhacs-operator
     operators.openshift.io/infrastructure-features: '["disconnected", "proxy-aware"]'
     support: Red Hat
+  labels:
+    operatorframework.io/arch.amd64: supported
```

Why don't we just add all architectures here unconditionally?

```suggestion
    operatorframework.io/arch.amd64: supported
    operatorframework.io/arch.ppc64le: supported
    operatorframework.io/arch.s390x: supported
```

Yes, the upstream operator does not come built for these architectures (today) but does this make anyone upset?

---

This is the other option.  However, it may be confusing, particularly if the community operator CSV gets imported into another index (e.g. OpenShift's community-operator-index).  Since you already have this script to modify the CSV, it doesn't seem hard to handle this as proposed here (style nits aside).

---

I would agree. The operator build process certainly isn't super elegant already, but this seems a fairly needless (as in: doing it at least somewhat better than that seems a fairly low-hanging fruit) breaking of abstraction.